### PR TITLE
ISPN-9520 JSON conversion causes NPE for events

### PR DESCRIPTION
**Pull request by gustavonalle**
https://issues.jboss.org/browse/ISPN-9520

### CODE REVIEW

File: client/hotrod-client/src/test/java/org/infinispan/client/hotrod/event/JsonEvents.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+import org.infinispan.client.hotrod.DataFormat;
+import org.infinispan.client.hotrod.test.SingleHotRodServerTest;
+import org.infinispan.commons.marshall.UTF8StringMarshaller;
+import org.testng.annotations.Test;
+
+/**
+ * @since 9.4
+ */
+@Test(groups = {"functional",}, testName = "client.hotrod.event.JsonEvents")
+public class JsonEvents extends SingleHotRodServerTest {
```
Comment (danberindei) [inline client/hotrod-client/src/test/java/org/infinispan/client/hotrod/event/JsonEvents.java]: Please name it `*Test` or surefire won't run it

Comment (gustavonalle) [reply client/hotrod-client/src/test/java/org/infinispan/client/hotrod/event/JsonEvents.java]: Done

**Comment by gustavonalle** [comment]
CI is saying ```Not enough free space (/dev/vda1       8.8G).```

**Comment by galderz** [comment]
Building this locally to see how it goes for Node.js client...

**Pull request merged**